checkov/runner_filter.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         all_external: bool = False,
         var_files: Optional[List[str]] = None,
         min_cve_severity: str = 'none',
-        skip_cve_package: Optional[List] = None
+        skip_cve_package: Optional[List[Any]] = None
```

Comment:
```suggestion
        skip_cve_package: Optional[List[str]] = None
```
would be surprised, if it is something else 😄